src/components/InvalidVinMessage/InvalidVinMessage.js

```diff
@@ -0,0 +1,19 @@
+import InvalidVinMessage from './StyledInvalidVinMessage';
+import Typography from '../../../styles/Typography';
+import useStore from '../../lib/hooks/useStore';
+
+export default function DisplayInvalidVinMessage() {
+	const displayVinError = useStore(state => state.displayVinError);
+
+	if (displayVinError) {
+		return (
+			<InvalidVinMessage>
+				<Typography variant="h4" color="white" textAlign="center">
+					VIN Eingabe inkorrekt! Bitte überprüfe deine VIN und probiere es erneut.
+				</Typography>
+			</InvalidVinMessage>
+		);
+	} else {
+		return null;
+	}
+}
```

Comment:
Your namings are pretty confusing:

You are importing StyledInvalidMessage as InvalidMessage into a component you import as InvalidMessage somewhere else but call DisplayInvalidMessage within.

Try not to rename you components on import, instead use the right naming in the beginning

Reply:
Fixed.